api/controllers/roomController.ts

```diff
@@ -0,0 +1,51 @@
+import { Request, Response } from "express";
+import roomService from "../services/roomService";
+
+class roomController {
+  static getRoomList = async (req: Request, res: Response) => {
+    const result = await roomService.getRoomList(req);
+    return res.status(200).json({ result });
+  };
+  static postRoom = async (req: Request, res: Response) => {
+    const { roomName, roomCategory } = req.body;
+    const userId = req.headers;
+    await roomService.postRoom(roomName, roomCategory, userId);
+    return res.status(201).json({ message: "SUCCESS_POST_ROOM" });
+  };
+
+  static getRoomInfo = async (req: Request, res: Response) => {
+    const result = await roomService.getRoomInfo(req);
+    return res.status(200).json({ result });
+  };
+
+  static patchOrderStatus = async (req: Request, res: Response) => {
+    const { roomId, orderStatus } = req.body;
+    await roomService.patchOrderStatus(orderStatus, roomId);
+    return res.status(201).json({ message: "SUCCESS_PATCH_ORDER_STATUS" });
```

Comment:
```suggestion
    return res.status(200).json({ message: "SUCCESS_PATCH_ORDER_STATUS" });
```
새로 생성하는게 아니니 201은 아닌 것 같습니다.

Reply:
복붙해오셨네요.